SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -313,6 +313,17 @@ void USpatialNetDriver::OnConnectionToSpatialOSFailed(uint8_t ConnectionStatusCo
 	}
 }
 
+void USpatialNetDriver::OnConnectionToSpatialOSFailed(uint8_t ConnectionStatusCode, const FString& ErrorMessage)
```

Comment:
Probably want to remove this as we already have the function implemented. I am pretty sure this was added to release just for comparability rather than a change in functionality.

Reply:
Yeah looks like a semantic merge conflict that didn't get written as a proper conflict when merging, nice catch will merge now.